packages/react-components/react-avatar/src/components/AvatarGroupPopover/useAvatarGroupPopover.tsx

```diff
@@ -77,14 +77,15 @@ export const useAvatarGroupPopover_unstable = (props: AvatarGroupPopoverProps):
     content: resolveShorthand(props.content, {
       required: true,
       defaultProps: {
-        'aria-label': 'Overflow',
         children: children,
-        role: 'list',
```

Comment:
I'd actually suggest leaving the `role: 'list'` because of an extremely niche Safari-specific issue with list styling